playground/DatabaseMigration/DatabaseMigration.AppHost/aspire-manifest.json

```diff
@@ -65,6 +51,24 @@
         "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true",
         "ConnectionStrings__db1": "{db1.connectionString}"
       }
+    },
+    "sql1-password": {
+      "type": "parameter.v0",
+      "value": "{sql1-password.inputs.value}",
+      "inputs": {
+        "value": {
+          "type": "string",
+          "secret": true,
+          "default": {
+            "generate": {
+              "minLength": 22,
+              "minLower": 1,
+              "minUpper": 1,
+              "minNumeric": 1
+            }
```

Comment:
@vhvb1989 - what if we made the `azd` rule that if a parameter has a `default`, we don't prompt. And a new optional property like `default.generate.manualPrompt: true` can put on the parameter to make `azd` prompt the user? That way by default, generated parameters just get generated. And only if someone opts in to prompting will it prompt.

Thoughts?

Reply:
having `allowManualPrompt` in `default.generate` sounds weird to me, because it would be next to the other rules, leading to questions like, what is allowManualPrompt=true + minLength=10.

I would add `allowManualPrompt` as a field for inputs.value

Reply:
I added support for azd for this with a field `NoManual` in the parameter metadata, here: https://github.com/Azure/azure-dev/pull/3580

So, a bicep parameter can be:
```bicep
@metadata({azd: {
  type: 'autoOrManualGeneration'
  config: {length:22}
  noManual:true
  }
})
@secure()
param pg1_password string
```

The noManual makes azd to skip the asking to customer and directly generating.

I can map any setting you add to the manifest to that azd feature